subgraph/schema.graphql

```diff
@@ -144,6 +144,19 @@ type Owner @entity {
   tokens: [Token!] @derivedFrom(field: "owner")
 }
 
+type Collection @entity {
```

Comment:
Should we add collection controllers in this same PR as well? Either that are we make a new card for it. The shortrcut title had controllers accounted for but the subtasks didn't so we missed it.  But let's decide if we want it in this PR or another one. 

Above that I also created a separate card for us to handle token owner and controller changes.

Reply:
I thought we are going to remove collection controllers per the discussion in the dev channel. Maybe @zoruka can confirm this, but since this role has no real special privileges, they are going to be removed in the ACL refactor.